ibis/backends/tests/test_string.py

```diff
@@ -171,6 +171,10 @@ def test_string_col_is_unicode(backend, alltypes, df):
             id='length',
             marks=pytest.mark.xfail_backends(['omniscidb']),  # #2338
         ),
+        param(lambda t: t.string_col.startswith('foo'),
+              lambda t: t.string_col.startswith('foo'), id='startswith'),
+        param(lambda t: t.string_col.endswith('foo'),
+              lambda t: t.string_col.endswith('foo'), id='endsswith'),
```

Comment:
This should be testing the SQLAlchemy implementation. If you want to be sure, you can break something on the SQLAlchemy function, and then run `PYTEST_BACKENDS="sqlite" python -m pytest ibis/backends/tests` and see if this test fails.

Reply:
Looks like the second function is working on a pandas DataFrame though?

Reply:
Is there a setup script to setup the databases? + I am getting errors of CSV files missing.